main.tf

```diff
@@ -153,15 +161,15 @@ resource "aws_iam_role" "ecs_task" {
   count = local.create_task_role ? 1 : 0
 
   name                 = module.task_label.id
-  assume_role_policy   = join("", data.aws_iam_policy_document.ecs_task.*.json)
+  assume_role_policy   = join("", data.aws_iam_policy_document.ecs_task[*].json)
```

Comment:
```suggestion
  assume_role_policy   = data.aws_iam_policy_document.ecs_task[0].json
```

Reply:
i don't see the sense of calling join for an array that can only consist of one element.